package.json

```diff
@@ -24,9 +24,11 @@
     "broken-link-checker": "^0.7.8",
     "clsx": "^1.1.1",
     "markdownlint-cli2": "^0.4.0",
+    "plotly.js": "^2.12.1",
```

Comment:
Hm, this is massive. It adds `Disk size with unique dependencies: "129.67MB"`

plotly.js-basic-dist would probably be equally good at `Disk size with unique dependencies: "2.72MB"`, and loaded with `createPlotComponent`

Reply:
Did you explore if that is possible?

Reply:
Nope

Reply:
But it has to be :D otherwise DELETE react-plotly.js